worlds/oot/__init__.py

```diff
@@ -983,3 +983,6 @@ def get_state_with_complete_itempool(self):
         all_state.stale[self.player] = True
 
         return all_state
+
+    def get_filler_item_name(self) -> str:
+        return get_junk_item(count=1)[0]
```

Comment:
This will not respect the modified junk fills imposed by the ice trap options. You should pass the result of `get_junk_pool(self)` as the `pool` argument to `get_junk_item`, so that you are drawing from the pools with increased ice traps.

Reply:
Done